packages/scripts/src/upgrade-tests/voting2/1_Propose.ts

```diff
@@ -98,7 +98,7 @@ async function main() {
 
   if (process.env[TEST_DOWNGRADE])
     votingUpgrader = await deployVotingUpgraderAndRunDowngradeOptionalTx(
-      (await hre.ethers.getSigners())[0].address,
+      process.env[EMERGENCY_EXECUTOR] || "",
```

Comment:
why would `_upgrader` argument be empty if no `EMERGENCY_EXECUTOR` was set?

Reply:
It's to fix a typescript issue and not adding a redundant check [existing here](https://github.com/UMAprotocol/protocol/blob/master/packages/scripts/src/upgrade-tests/voting2/migrationUtils.ts#L104)

But you are right, it doesn't look very solid. 

Fixed [here](https://github.com/UMAprotocol/protocol/pull/4459/commits/1966bc43e27fe7af93fd1dcbbce6fb1f9de90285)